android/quest/src/main/assets/configs/app/application_config.json

```diff
@@ -38,5 +38,48 @@
       "List",
       "Flag"
     ]
-  }
+  },
+  "eventWorkflows": [
+    {
+      "eventType": "RESOURCE_CLOSURE",
+      "eventResources": [
+        {
+          "id": "sickChildConditionToBeClosed",
+          "resource": "Condition",
+          "configRules": [
+            {
+              "name": "patientId",
+              "condition": "true",
+              "actions": [
+                "data.put('patientId', fhirPath.extractValue(CarePlan, 'CarePlan.subject.reference'))"
+              ]
+            }
+          ],
```

Comment:
Do we need config rules here?

Reply:
Yes. we need a way to extract the patient Id from the Careplan.

Reply:
Oh I see